homeassistant/components/binary_sensor/workday.py

```diff
@@ -59,6 +60,7 @@
     vol.Optional(CONF_PROVINCE): cv.string,
     vol.Optional(CONF_WORKDAYS, default=DEFAULT_WORKDAYS):
         vol.All(cv.ensure_list, [vol.In(ALLOWED_DAYS)]),
+    vol.Optional(CONF_ADDHOLIDAYS): vol.All(cv.ensure_list),
```

Comment:
We should also validate that the entries are strings.

Reply:
added to line vol.All(cv.ensure_list, **cv.string**), let me know if this is what you were looking for